roles/openshift_persistent_volumes/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Create temp directory for volume definitions
+  command: mktemp -d /tmp/openshift-ansible-XXXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Deploy PersistentVolume definitions
+  template:
+    dest: "{{ mktemp.stdout }}/persistent-volumes.yml"
+    src: persistent-volume.yml.j2
+  changed_when: False
+
+- name: Create PersistentVolumes
+  command: >
+    {{ openshift.common.client_binary }} create -f {{ mktemp.stdout }}/persistent-volumes.yml
```

Comment:
We should get away from using the default kubeconfig..  since you are already creating a temporary directory for the volume defs, copy /etc/origin/master/admin.kubeconfig there and use the temp copy instead.